src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -390,6 +415,7 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
     } else {
       this._eventManager.listenOnce(this._videoElement, Html5EventType.CAN_PLAY, parseTracksAndResolve.bind(this));
     }
+    this._startTimeAttach = null;
```

Comment:
```suggestion
    this. _lastTimeDetach = null;
```
actuaally i would reset it to `0` and type it as `number`